### DISCUSSION TITLE
How can I nest code blocks?

### DISCUSSION THREAD

**Discussion by Scotty78211** [squidfunk/mkdocs-material#5819] [Q&A] [answered]
It is used in many examples in the documentation, However It is not mentioned how to do it.

![image](https://github.com/squidfunk/mkdocs-material/assets/76183029/e8512f41-fff8-4339-b3ea-b8710137cfb5)

As it is showed here, the code block and below it's output as a nested code block.

**Comment by lampensau** [comment, marked as answer]
Try using [custom CSS](https://squidfunk.github.io/mkdocs-material/customization/) and [HTML](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/?h=html#markdown-in-html) to achieve what you want. The official documentation [does](https://github.com/squidfunk/mkdocs-material/blob/master/docs/reference/code-blocks.md?plain=1#L205-L226) the same:

````
<div class="result" markdown>

``` py
import tensorflow as tf
```

</div>
````

**Reply by Scotty78211** [reply]
There is no need to add a cutom CSS, it's already available.

<pre>
```` markdown title="Code block"
``` py
 import tensorflow as tf
```
````

&lt;div class="result" markdown&gt;
``` py
import tensorflow as tf
```
&lt;/div&gt;
</pre>

![image](https://github.com/squidfunk/mkdocs-material/assets/76183029/fac9ede1-5533-4fd1-80ad-e338659b4373)

I copy past it from the GitHub page you shared and it is working perfectly. Thanks a lot :thumbsup:
